bucket/shmnview.json

```diff
@@ -22,7 +22,7 @@
             "hash": "bd8d233e44155100677f855b61b867b596d2006bd65b4553da6ca5e38f70d483"
         }
     },
-    "description": "Disable/enable static menu items in context menu of Explorer.",

```

Comment:
Why is this an issue? These are pulled from nirsoft.com, so we can't really edit them. We'll have to [hack](https://github.com/ScoopInstaller/Nirsoft/blob/82cd8940540c6a8aa5f2b9c41e7030219d423d5d/update.py#L322) the update script instead.

Reply:
Ah, I didn't know. It's not really a problem as such, just that it makes the species look weird (slashes are replaced with underscores when the shortcut is created).

Reply:
I’ll close this, but if you wanna submit a PR, I’ll accept it. Thx.